#### QUICK START

***
##### 数据准备
```
# get ppi_data.py
apt-get update && apt-get install -y curl
# 创建/data
mkdir /data
cd /data
curl -k -O https://raw.githubusercontent.com/alibaba/euler/master/examples/ppi_data.py
pip install networkx==1.11 sklearn -i https://pypi.tuna.tsinghua.edu.cn/simple
```

##### 数据获取
`python ppi_data.py`
```
Traceback (most recent call last):
  File "ppi_data.py", line 32, in <module>
    from euler.tools import json2dat
  File "/usr/local/lib/python2.7/dist-packages/euler/__init__.py", line 20, in <module>
    from euler.python.service import *
  File "/usr/local/lib/python2.7/dist-packages/euler/python/service.py", line 27, in <module>
    _LIB = ctypes.CDLL(_LIB_PATH)
  File "/usr/lib/python2.7/ctypes/__init__.py", line 366, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libhdfs.so.0.0.0: cannot open shared object file: No such file or directory****

# ERROR: OSError!!!
```
```
# 环境变量缺失，重新export
export JAVA_HOME=/usr/lib/jvm/default-java
export LD_LIBRARY_PATH=$JAVA_HOME/lib/server:$LD_LIBRARY_PATH
export HADOOP_HOME=/usr/local/hadoop-2.9.2
export LIBRARY_PATH=$HADOOP_HOME/lib/native:$LIBRARY_PATH
export LD_LIBRARY_PATH=$HADOOP_HOME/lib/native:$LD_LIBRARY_PATH
export CLASSPATH=$($HADOOP_HOME/bin/hadoop classpath --glob):$CLASSPATH

# download ppi data..
```

